opal/mca/accelerator/accelerator.h

```diff
@@ -306,12 +306,9 @@ typedef struct {
  * We assume that only one accelerator will be present
  * on any given node.
  *
- * @param[OUT]  priority     Relative priority or ranking used by MCA to
- *                           select a component.
  * @return                   Initialized module or NULL if init failed.
  */
-typedef opal_accelerator_base_module_t * (*mca_accelerator_base_component_init_fn_t)(
-    int *priority);
```

Comment:
So reading over the selection logic, I think it's more confusing than what was previously there. There is an unnecessary amount of list manipulation imo. Why not just use the priority code that existed and return a fatal error if the initialized list length > 2?

Reply:
discussed offline, fine as is